build.yaml

```
@@ -1,9 +1,9 @@
 name: "TVHeadend"
 guid: "3fd018e5-5e78-4e58-b280-a0c068febee0"
 imageUrl: "https://repo.jellyfin.org/releases/plugin/images/jellyfin-plugin-tvheadend.png"
-version: 11
-targetAbi: "10.8.0.0"
-framework: "net6.0"
+version: 12
```

```suggestion
version: 11
```

Version bump is handled automatically